static/app/components/container/accordion.tsx

```diff
@@ -67,6 +67,10 @@ const AccordionHeader = styled('div')`
   align-items: center;
   border-top: 1px solid ${p => p.theme.border};
   padding: ${p => p.theme.space.md} ${p => p.theme.space.xl};
+  margin-bottom: ${p =>
+    p.theme.space
+      .sm}; /* Compensate for the border so it doesn't cut into previous item's padding */
+
```

Comment:
woah, strange line breaks. does it like this?

```suggestion
  /* Margin to compensate for the border so it doesn't cut into previous item's padding */
  margin-bottom: ${p => p.theme.space.sm}; 
```